tracks/go/exercises/pig-latin/mentoring.md

```diff
@@ -0,0 +1,72 @@
+# Pig Latin
+
+### Reasonable solutions:
```

Comment:
This should probably be `##` and not `###` based on a spot check of other notes

Reply:
Yeah, the other notes in the Go track seem to disagree with [the README for the repo](https://github.com/exercism/website-copy/blob/main/README.md?plain=1#L24), which suggests `###` for headings. I wasn't sure which to follow.